MediaBrowser.Providers/Subtitles/SubtitleManager.cs

```diff
@@ -19,6 +19,7 @@
 using MediaBrowser.Model.IO;
 using MediaBrowser.Model.Providers;
 using Microsoft.Extensions.Logging;
+using static MediaBrowser.Model.IO.StreamDefaults;
```

Comment:
why is this suddenly needed?

Reply:
For the ‘DefaultFileStreamBufferSize’ constant